numpy/core/tests/test_deprecations.py

```diff
@@ -504,3 +504,12 @@ class TestGeneratorSum(_DeprecationTestCase):
     # 2018-02-25, 1.15.0
     def test_generator_sum(self):
         self.assert_deprecated(np.sum, args=((i for i in range(5)),))
+
+class TestSctypeNA(_VisibleDeprecationTestCase):
+    # 2018-06-24, 1.16
+    def test_sctypeNA(self):
+        def show(_dict, key):
+            return _dict[key]
+        self.assert_deprecated(show, args=(np.core.sctypeNA, '?',))
+        self.assert_deprecated(show, args=(np.core.typeNA, '?',))
```

Comment:
Not sure this is better than just `self.assert_deprecated(lambda: np.sctypeNA['?'])`

Reply:
adopted